packages/edit-site/src/components/global-styles/style.scss

```diff
@@ -51,6 +51,7 @@
 	}
 }
 
+.edit-site-global-styles-header__hint,
```

Comment:
It may be worth exploring whether a small font would make sense. I am not sure how that fits with our design approach for the side panel, but I think it's worth considering.